README.md

```diff
@@ -52,13 +52,16 @@ The `pyproject.toml` configuration file can be and is also used to store 3rd par
 Install the project and its poetry dependencies, by using:
```

Comment:
maybe add sth like: `make sure you have this python version installed, or otherwise, try with a different python version at your own risk`